pages/tidal tools/sourcecode.md

```diff
@@ -11,67 +11,34 @@ folder: tidaltools
 
 Not sure how ready you are to move to the cloud? With Tidal you have
 the option to analyze your specified source code associated with the applications.
-
 The analysis will identify the difficulty to migrate your applications to the
-cloud, including the number of blocking issues identified. It is able to
```

Comment:
I understand we have to remove what we don't have now, and later add-back explicitly what we do support. All good.

For inspiration, I wanted to leave this reference to [supported languages](https://github.com/AlDanial/cloc#recognized-languages-) and [custom language definitions](https://github.com/AlDanial/cloc#create-custom-language-definitions-) with CLOC right here.